code/game/objects/items/shields.dm

```diff
@@ -153,3 +153,15 @@
 		slot_flags = null
 		to_chat(user, "<span class='notice'>[src] can now be concealed.</span>")
 	add_fingerprint(user)
+
+/obj/item/shield/makeshift
+	name = "metal shield"
+	desc = "A large shield made of wired and welded sheets of metal. The handle is made of cloth and leather making it unwieldy."
+	armor = list("melee" = 25, "bullet" = 25, "laser" = 5, "energy" = 0, "bomb" = 30, "bio" = 0, "rad" = 0, "fire" = 70, "acid" = 80)
+	lefthand_file = 'icons/mob/inhands/equipment/shields_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/shields_righthand.dmi'
+	item_state = "makeshift_shield"
+	slot_flags = null
+	block_chance = 20

```

Comment:
overall its block_chance is even lower than a buckler, and takes even more effort to make due to the leather requirements.